pkg/kv/kv_object_format.go

```diff
@@ -0,0 +1,54 @@
+package kv
+
+import (
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+var matchers = make(map[string]protoreflect.ProtoMessage)
+var defaultMsg protoreflect.ProtoMessage = nil
```

Comment:
I don't think we need a default msg - if we can't find a prefix to decode the proto we can decode the message dynamically. Print out for each field the code/type/value.

Reply:
1. I'm not sure I follow - don't we need to have at least some info on the msg type? Otherwise, how can the different fields be differentiate. Can you refer me to some example on how it can be done?
2. Why not use a default type? There is currently only one path we cannot identify correctly

Reply:
My problem is that the default type solve one problem which is indication for the next one...
We set the default to staging object proto - which means we don't have a pattern that match this type.
It looks like based on the pattern of partition and/or path we should be able to identify the type, so the fallback will be the raw value and not proto message.
There is a solution to dynamically dump any protobuf message, but as I see it now it is not required.